src/main/java/com/icthh/xm/ms/entity/repository/XmEntityRepository.java

```diff
@@ -179,14 +211,16 @@ public XmEntity findOne(Long aLong, List<String> embed) {
         return springXmEntityRepository.findAll(jpql, args, embed);
     }
 
-    public <S extends XmEntity> List<S> save(Iterable<S> entities) {
+    @Override
+    public <S extends XmEntity> List<S> saveAll(Iterable<S> entities) {
```

Comment:
It's can broke leps!
Please rollback rename method.

Reply:
done